src/operations/bulk_write.ts

```diff
@@ -50,11 +46,8 @@ export class BulkWriteOperation extends OperationBase<BulkWriteOptions, BulkWrit
       return callback(err);
     }
 
-    // Final options for retryable writes and write concern
     let finalOptions = Object.assign({}, options);
     finalOptions = applyRetryableWrites(finalOptions, coll.s.db);
-    finalOptions = applyWriteConcern(finalOptions, { db: coll.s.db, collection: coll }, options);
-
     const writeCon = WriteConcern.fromOptions(finalOptions);
```

Comment:
why was this `writeCon` left here? It seems like it should be otherwise solved through options resolution

Reply:
The options resolution ensures that `options` has the right `writeConcern`, but in this case we still have to pull the `WriteConcern` object from the options to pass to `bulk.execute(writeConcern, options, callback)`. Happy to remove this argument from `bulk.execute(...)`, since it could just be pulled from the options within the method.

Reply:
oh yeah, let's definitely do that!